packages/react/src/components/shared/__tests__/ValidationErrors.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { ValidationErrors } from '../ValidationErrors';
+
+describe('ValidationErrors', () => {
+  it('renders errors as expected', async () => {
+    const { container } = render(
+      <ValidationErrors errors={['mock error 1', 'mock error 2']} />
+    );
+    expect(container).toMatchSnapshot();
+    expect(await screen.findByText('mock error 1')).toBeDefined();
+    expect(await screen.findByText('mock error 2')).toBeDefined();
+  });
+
+  it('renders nothing if there are no errors', async () => {
+    const { container } = render(<ValidationErrors errors={[]} />);
+    expect(container).toMatchInlineSnapshot(`<div />`);
```

Comment:
Shouldn't it return `null` if there are no errors?

Reply:
Yeah, I was going back and forth about this. I was initially thinking about having `Authenticator` or `AccountSettings` handle empty array and make it `null`, but I think it makes sense to just return `null` here.

Adjusting.

Reply:
Updated in [4e570a5](https://github.com/aws-amplify/amplify-ui/pull/2908/commits/4e570a53152c6c7469fda6088caff3370b75491f). Added one unhappy unit test to `defaultService.test.ts` to feel safe too.